WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BottomSheetNoticeView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+struct BottomSheetNoticeView: View {
+    @Environment(\.presentationMode) private var presentation
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var buttonTapped: (() -> Void)?
+    var title: String
+    var message: String
+    var confirmationButtonMessage: String
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Spacer()
+            Text(title)
+                .font(.headline)
+                .padding(.bottom, Layout.verticalPadding)
+            HStack(alignment: .center, spacing: 16.0) {
+                Text(message)
+                    .font(.caption)
+                    .foregroundColor(.gray)
+                    .multilineTextAlignment(.leading)
+                    .fixedSize(horizontal: false, vertical: true)
+                Spacer()
+                Image(uiImage: .walletImage)
```

Comment:
This image is hardcoded, we can make it injectable from the outside to keep it generic :)